### PR TITLE
DAT-2857 Horizontal group

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/DAT-2857

### CODE REVIEW

File: extensions/wikia/PortableInfobox/PortableInfobox.setup.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	'name' => 'Portable Infobox',
 	'author' => [
 		'Adam Robak',
+		'Diana Falkowska',
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/PortableInfobox.setup.php]: :smiling_imp:

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeGroup.php [hunk 2]
```diff
@@ -4,13 +4,23 @@
 use Wikia\PortableInfobox\Parser\XmlParser;
 
 class NodeGroup extends Node {
+	const DATA_LAYOUT_ATTR_NAME = 'layout';
+	const GROUP_LAYOUT_PREFIX = 'group-layout-';
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeGroup.php]: Where is it used?

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php [hunk 3]
```diff
@@ -119,6 +119,14 @@ protected function getXmlAttribute( \SimpleXMLElement $xmlNode, $attribute ) {
 			: null;
 	}
 
+	protected function getXmlAttributeFromSupported( \SimpleXMLElement $xmlNode, $attribute, $supportedAttributes ) {
+		$attr = $this->getXmlAttribute( $xmlNode, $attribute );
+		if ( isset($attr) && in_array( $attr, $supportedAttributes ) ) {
+			return $attr;
+		}
+		return self::DEFAULT_TAG_NAME;
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: you probably will want static::DEFAULT_TAG_NAME to benefit fully from late static binding

**Comment by RafLeszczynski** [comment]
LGTM but please run scss-lint and fix any reported issues

**Comment by SebastianMarzjan** [comment]
Go for it

**Pull request merged**